store/sqlstore/store.go

```diff
@@ -137,11 +138,18 @@ type SqlStoreStores struct {
 type SqlStore struct {
 	// rrCounter and srCounter should be kept first.
 	// See https://github.com/mattermost/mattermost-server/v6/pull/7281
-	rrCounter         int64
-	srCounter         int64
-	master            *gorp.DbMap
-	Replicas          []*gorp.DbMap
-	searchReplicas    []*gorp.DbMap
+	rrCounter int64
+	srCounter int64
+
+	master  *gorp.DbMap
+	masterX *sqlx.DB
+
+	Replicas  []*gorp.DbMap
+	ReplicaXs []*sqlx.DB
+
+	searchReplicas  []*gorp.DbMap
+	searchReplicaXs []*sqlx.DB
```

Comment:
Is the plan to keep both (gorp and sqlx) for a while or just during development?

Reply:
Yes, the plan is to keep them both during the transition period. And then remove them, and change all (Master|Replica)X methods to (Master|Replica) ones.

Reply:
Sounds good :+1: